package.json

```diff
@@ -14,7 +14,7 @@
     "coverage": "yarn test --coverage",
     "analyze": "webpack --mode=production --env.analyze=true",
     "prepare": "husky install",
-    "extract-translations": "i18next 'src/**/*.component.tsx' --config './tools/i18next-parser.config.js'"
+    "extract-translations": "i18next 'src/**/*.{ts,tsx}' --config './tools/i18next-parser.config.js'"
```

Comment:
Did you intend to make this change?

Reply:
yes this was intended

Reply:
You don't need this since the translations will be done by the form-engine-app